finagle-redis/src/main/scala/com/twitter/finagle/Redis.scala

```diff
@@ -63,4 +66,48 @@ object Redis extends Client[Command, Reply] {
 
   def newService(dest: Name, label: String): Service[Command, Reply] =
     client.newService(dest, label)
+
+  object Subscribe extends com.twitter.finagle.Client[SubscribeCommand, Unit] {
```

Comment:
could you document why we need a special subscription client, here and elsewhere?  presumably it's because finagle doesn't have native support for object-streaming-style computation.  it's also worth noting that if instead of a Service[Req, Rep] we had a Client that could return a Closable AsyncStream, that would probably be better, and it's something we should probably target as we try to retool finagle to support more streaming work.